### PR TITLE
Videntis Shrine Lines Review

### PR DESCRIPTION
**Pull request by ttgmichael**
Attempt at translating the lines in Videntis Shrine.

Thanks TentuZero for the screenshots as reference material!

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -11773,17 +11773,17 @@ ETC_20150714_011774	Magic Field of Preservation
...
-ETC_20150714_011780	It seems that you've been spotted while you were destroying the pots of silence!{nl}Defeat all the demons that are rushing in!
-ETC_20150714_011781	The pots of silence
-ETC_20150714_011782	Protect until the contract field of the bodies all burn out!
-ETC_20150714_011783	Contract Field of the Bodies
-ETC_20150714_011784	Thanks. Now, they can go near the goddesses.
-ETC_20150714_011785	May the goddesses bless you in the future.
-ETC_20150714_011786	Defeat Master Gini and release the contract of the spirit!
-ETC_20150714_011787	Holy sphere has been activated!
-ETC_20150714_011788	Block against the attacks from subordinates of Naktis until the transformation scroll gets completed!
+ETC_20150714_011778	Sealed Spatial Magic Stone
```
Comment (Sunaries) [inline ETC.tsv]: Stone ? You also call it 'gem' in other lines.

**Comment by Sunaries** [comment]
@ttgmichael 

Can you add the following changes to this PR, please?
1. SKILL_20150717_004835    $Decreases the cooldown time of [Carve Vakarine] to 10 seconds. -> $Decreases the cooldown time of [Carve Vakarine] by 10 seconds.
2. SKILL_20150717_004744    $Enemies affected by [Stun] because of [Thrust] will be afflicted with [Bleeding] for 5 seconds. Bleeding damage depends on your STR.

**to**

Enemies affected by [Stun] because of [Thrust] will be afflicted with [Bleeding] for 5 seconds. Bleeding damage is proportional to the character's STR.

**or**

Afflicts [Bleeding] for 5 seconds on stunned enemies from [Thrust]. Bleeding damage is proportional to the character's STR.

I vote for the 2nd description since its better.

**Comment by Sunaries** [comment]
:+1:

**Comment by ttgmichael** [comment]
:+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**